sdk/keyvault/keyvault-common/review/keyvault-common.api.md

```diff
@@ -0,0 +1,41 @@
+## API Report File for "@azure/keyvault-common"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { ChallengeCallbacks } from '@azure/core-rest-pipeline';
+
+// @public
+export function createChallengeCallbacks(options?: CreateChallengeCallbacksOptions): ChallengeCallbacks;
+
+// @public
+export interface CreateChallengeCallbacksOptions {
+    disableChallengeResourceVerification?: boolean;
+}
+
+// @public
+export interface ParsedKeyVaultEntityIdentifier {
+    name: string;
+    vaultUrl: string;
+    version?: string;
+}
+
+// @public
+export interface ParsedWWWAuthenticate {
```

Comment:
Could the name be simplified to `WWWAuthenticate`?

Reply:
This isn't part of the API surface anymore (since `parseWWWAuthenticate` isn't anymore either), but I agree we can simplify this. Went ahead with the rename.

I think the same argument applies to the still-exported `ParsedKeyVaultEntityIdentifier`. I've simplified that to just `KeyVaultEntityIdentifier` -- the other SDKs narrow down that type further in actual use.

Reply:
Awesome, thank you sir!